cfme/tests/test_replication.py

```diff
@@ -322,7 +337,22 @@ def test_replication_global_to_remote_new_vm_from_template():
             2.
             3. VM created in the Remote, no errors.
     """
-    pass
+    remote_app, global_app = setup_replication
+    remote_provider = provider_app_crud(RHEVMProvider, remote_app)
+    remote_provider.setup()
+    assert remote_provider.name in remote_app.managed_provider_names, "Provider is not available."
+
+    new_vm_name = "test_replication_global_to_remote_new_vm_from_template"
```

Comment:
I think the VM name should be randomized, in case cleanup fails for some reason.